Source/Scene/Model.js

```diff
@@ -1077,8 +1079,35 @@ define([
                             var position = primitives[m].attributes.POSITION;
                             if (defined(position)) {
                                 var accessor = gltfAccessors[position];
-                                var aMin = Cartesian3.fromArray(accessor.min, 0, aMinScratch);
-                                var aMax = Cartesian3.fromArray(accessor.max, 0, aMaxScratch);
+                                var extensions = accessor.extensions;
+                                var accessorMin = new Matrix(accessor.min);
+                                var accessorMax = new Matrix(accessor.max);
+                                
+                                // If this accessor is quantized, we should use the decoded min and max
+                                if (defined(extensions)) {
+                                    var quantizedAttributes = extensions.WEB3D_quantized_attributes;
+                                    if (defined(quantizedAttributes)) {
```

Comment:
What is the rationale in the extension spec for storing min/max quantized?  Do you think it is the right approach?  What is the benefit?  Size is not important here.  Won't almost all clients just decode it right away?